CONTRIBUTING.md

```diff
@@ -51,6 +51,26 @@ yarn workspaces foreach -Rt --no-private run build
 
 - `yarn reset` - will clean all `node_modules`, re-install dependencies and link them, and then build all JSS packages.
 
+## Developing sample
+
+For the creation of the samples we are using `initializers` approach. 
+
+- `initializer` - the process for handling the creation of the appropriate template. See [initializers](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss/src/initializers).
+- `template` - the sample populated by [ejs](https://ejs.co/) tokens. See [templates](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss/src/templates). Templates can be:
+	- `base` - the template that contains foundation for the application (e.g. *nextjs*).
+	- `feature` - the template that provides specific feature for the base template. Multiple *feature* templates can be applied to the *base* (e.g. *nextjs-styleguide*).
+
+If you want to use [*create-sitecore-jss*](https://github.com/Sitecore/jss/tree/dev/packages/create-sitecore-jss) from your local repository, run:
+
+```
+cd ./packages/create-sitecore-jss
+npm i -g
+```
```

Comment:
add `npm link` step

Reply:
We don't need `npm link` because when you run `npm i -g` it will install from this directory

Reply:
@CobyPear So when you will run `npm init sitecore-jss` - local package should be used

Reply:
Ah I totally missed the -g. Makes sense!